logdna/lib/cloudWatch.js

```diff
@@ -0,0 +1,34 @@
+// External Libraries
+const zlib = require('zlib');
+
+// Parse the GZipped Log Data
+const parseEvents = (event) => {
+    return JSON.parse(zlib.unzipSync(Buffer.from(event.awslogs.data, 'base64')));
```

Comment:
I have seen in lots of places where `JSON.parse` failed because the string is not in JSON format. Shall we start wrapping this in `try...catch...` block and do some logging instead of failing the client with an exception?

Reply:
I think it's a good idea.

Reply:
Actually, I don't think we can log it. I did not see in AWS the printed out logs. So I think it is actually fail with the error here. @smusali  Do you know if AWS can show the execution logs?

Reply:
You can pass the error thru the `callback` defined [here](https://github.com/logdna/logdna-cloudwatch/blob/master/logdna/index.js#L132)

Reply:
I think I tried doing that when played with it and I did not see my messages displayed anywhere =) Maybe I looked at a wrong place.